src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represent a Person's contact in the address book.
+ */
+public class Contact {
+
+	public final String value;
```

Comment:
try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.